### ISSUE TITLE
Remember fullscreen state and window size

### ISSUE THREAD

**Issue by MainMemory** [blupi-games/planetblupi#16]
Currently, if you change the settings ingame to windowed mode or if you change the window size, it doesn't remember the settings when you start the game the next time.

**Comment by Skywalker13** [comment]
Yes, it was a deliberate choice.

You can force the fullscreen mode with the config.json file in the data directory (when installed).  
https://github.com/blupi-games/planetblupi/blob/master/README-user.md#screen-problems

**Comment by Skywalker13** [comment]
Other way, you can start the game with `planetblupi.exe --fullscreen off`.

**Comment by MainMemory** [comment]
That doesn't do anything about the window size though.

**Comment by Skywalker13** [comment]
I can add an option and a setting for the window size for the next patch version

**Comment by Skywalker13** [comment]
The window size can be changed (for the v1.9.3, when it will be released) by using:
```sh
planetblupi --fullscreen off --zoom 2
# or
planetblupi -f off -z 2
```

And/or with the `config.json` file:
```json
{
  "fullscreen": false,
  "zoom": 2
}
```

**Comment by Skywalker13** [comment]
This setting and command line argument is available with `v1.10` on www.blupi.org
